delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/CompositeSink.scala

```diff
@@ -123,6 +123,7 @@ final class Batch[SinkFormat](
   /** Replaces the graph of a provided [[GraphResource]] by extracting its new graph from the provided (full) graph. */
   private def replaceGraph(gr: GraphResource, fullGraph: Graph) = {
     implicit val api: JsonLdApi = JsonLdJavaApi.lenient
+    implicit val options        = JsonLdOptions.AlwaysEmbed
```

Comment:
I think putting it here means it would only apply to the `BatchSink` and not the `SingleSink`.